tutorial/start.nim

```diff
@@ -0,0 +1,33 @@
+when not(compileOption("threads")):
+  {.fatal: "Please, compile this program with the --threads:on option!".}
+  
+import chronos          # an efficient library for async, developed by Status
```

Comment:
we don't need the "developed by Status" bit really - instead, we tend to put a license header on top